### PR TITLE
Add kubectl patch example with quotes on Windows

### PR DESCRIPTION
**Pull request by merkata**
When running the `kubectl patch` example, on Windows systems you get an
error when passing the patch request in single quotes. Passing it in
double quotes with the inner ones escaped produced the desired behavior
as is in the example given for Linux systems. I've added a small note
for Windows users to have that in mind.

Signed-off-by: Mariyan Dimitrov <mariyan.dimitrov@gmail.com>

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/change-pv-reclaim-policy.md [hunk 1]
```diff
@@ -54,6 +54,13 @@ the corresponding `PersistentVolume` is not be deleted. Instead, it is moved to
 
     where `<your-pv-name>` is the name of your chosen PersistentVolume.
 
+    *Note* if you are running `kubectl` on Windows, you'd need to use double quotes on the
+    command line and thus escape the inner double quotes like this:
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/change-pv-reclaim-policy.md]: Would you mind altering that text to use a `{{< note >}}` [shortcode](https://kubernetes.io/docs/contribute/style/style-guide/#shortcodes)? That fits in better with the website style guidelines.

Comment (merkata) [reply content/en/docs/tasks/administer-cluster/change-pv-reclaim-policy.md]: Thanks @sftim , I've placed the note inside a shortcode.

File: content/en/docs/tasks/administer-cluster/change-pv-reclaim-policy.md [hunk 2]
```diff
@@ -54,6 +54,15 @@ the corresponding `PersistentVolume` is not be deleted. Instead, it is moved to
 
     where `<your-pv-name>` is the name of your chosen PersistentVolume.
 
+    {{< note >}}
+    If you are running `kubectl` on Windows, you'd need to use double quotes on the
+    command line and thus escape the inner double quotes like this:
+
+    ```shell
+    kubectl patch pv <your-pv-name> -p "{\"spec\":{\"persistentVolumeReclaimPolicy\":\"Retain\"}}"
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/change-pv-reclaim-policy.md]: Maybe remove the three-backtick lines before and after, and change this to:
`kubectl patch pv <your-pv-name> -p "{\"spec\":{\"persistentVolumeReclaimPolicy\":\"Retain\"}}"`

Comment (merkata) [reply content/en/docs/tasks/administer-cluster/change-pv-reclaim-policy.md]: Thanks @sftim , I actually noticed the page broke after the last change, so I started testing locally with the kubernetes-hugo docker image. I'll see how it looks without the backticks and if the page remains in tact and will push the latest change afterwards, thanks!

Comment (merkata) [reply content/en/docs/tasks/administer-cluster/change-pv-reclaim-policy.md]: @sftim it renders correctly now with the latest commit

File: content/en/docs/tasks/administer-cluster/change-pv-reclaim-policy.md [hunk 3]
```diff
@@ -54,6 +54,15 @@ the corresponding `PersistentVolume` is not be deleted. Instead, it is moved to
 
     where `<your-pv-name>` is the name of your chosen PersistentVolume.
 
+    {{< note >}}
+    On Windows, you must _double_ quote any JSONPath template that contains spaces (not single quote as shown above for bash). This in turn means that you must use a single quote or escaped double quote around any literals in the template. For example:
+
+```cmd
+C:\> kubectl patch pv <your-pv-name> -p "{\"spec\":{\"persistentVolumeReclaimPolicy\":\"Retain\"}}"
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/change-pv-reclaim-policy.md]: As per the style guide, you should [avoid putting the prompt](https://kubernetes.io/docs/contribute/style/style-guide/#don-t-include-the-command-prompt) in sample command-line blocks.
```suggestion
kubectl patch pv <your-pv-name> -p "{\"spec\":{\"persistentVolumeReclaimPolicy\":\"Retain\"}}"
```
If you need to emphasize the context, I'd do that in a comment line above the sample command.

Comment (merkata) [reply content/en/docs/tasks/administer-cluster/change-pv-reclaim-policy.md]: Agree, I've changed it according to the style guide and also made another PR against reference/kubectl/jsonpath.md, as I've taken the example with the command prompt from there. Thanks @sftim!

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-18853--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-cluster/change-pv-reclaim-policy/

**Pull request merged**